packages/sdds-icons/stories/IconGallery.css

```
@@ -0,0 +1,68 @@
+    background: #fafafa;
+}
+
+.icon-gallery__table img {
+    display: block;
+    margin: 0 auto 8px;
+}
+
+.icon-gallery__name {
+    display: block;
+    overflow-wrap: anywhere;
+    font-family: 'SFMono-Regular', Consolas, 'Liberation Mono', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)